client/layout/logged-out.jsx

```diff
@@ -38,8 +38,8 @@ const LayoutLoggedOut = ( {
 	useOAuth2Layout,
 } ) => {
 	const classes = {
-		[ 'is-group-' + section.group ]: !! section,
-		[ 'is-section-' + section.name ]: !! section,
+		[ 'is-group-' + section.group ]: !! ( section && section.group ),
+		[ 'is-section-' + section.name ]: !! ( section && section.name ),
```

Comment:
Maybe use (lodash's) `get( section, 'group', false )` (and `name`, respectively) which does essentially just this?

Reply:
Afaik `lodash.get` only evaluates, in this case, to `false` if the property is `undefined`. Which means this

```
> obj
{ a: 1, b: null }
> lodash.get(obj, 'b', false)
null
```

In general this would be fair enough but we are also casting those values to a Boolean which is not covered by lodash.get.

Or didn't you mean omitting the `!!`? In this case we could just replace it but don't need to pass in `false` I guess?

Reply:
Actually, I might have missed your point as classnames accepts truthy/falsy values. In this case we don't even need the `!!` and can omit the `false`, true?

So `get( section, 'group' )` should be sufficient.